Kickstarter-iOS/Views/CommentComposerView.swift

```diff
@@ -0,0 +1,140 @@
+import Library
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum Border {
+    static let height: CGFloat = 1.0
+  }
+
+  enum Avatar {
+    static let diameter: CGFloat = 44.0
+    static let margin: CGFloat = 18.0
+  }
+}
+
+final class CommentComposerView: UIView {
+  // MARK: - Properties
+
+  private let rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+
+  private let topBorderView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let inputContainerView: CommentInputContainerView = {
+    CommentInputContainerView()
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private lazy var avatarImageView = {
+    CircleAvatarImageView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let onlyBackersLabel: UILabel = {
+    UILabel(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  // This will be replaced with a ViewModel ouput binding
```

Comment:
Let's prefix all of these sorts of TODO's with `// TODO:` so that they're easier to find later on.